src/main/resources/config/liquibase/changelog/20211029230000_changelog.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="imstreetspirit" id="20211029230000">
+        <!--SQL script that updates detail_text and text column of feedback table.-->
+        <!--It finds the feedback which is associated with grading instruction. Detail_text or text column might contain
```

Comment:
can you please move the comments inline so that they are directly above the corresponding queries?